components/style/values/specified/length.rs

```diff
@@ -282,6 +331,31 @@ impl AbsoluteLength {
             | AbsoluteLength::Pc(v) => v == 0.,
         }
     }
+
+    /// Convert this into a pixel value.
+    #[inline]
+    pub fn to_px(&self) -> CSSFloat {
+        use std::f32;
+
+        const PX_PER_IN: CSSFloat = 96.;
+        const PX_PER_CM: CSSFloat = PX_PER_IN / 2.54;
+        const PX_PER_MM: CSSFloat = PX_PER_IN / 25.4;
+        const PX_PER_Q: CSSFloat = PX_PER_MM / 4.;
+        const PX_PER_PT: CSSFloat = PX_PER_IN / 72.;
+        const PX_PER_PC: CSSFloat = PX_PER_PT * 12.;
```

Comment:
Can we remove the `AU_PER_FOO` constants and at least reuse this? If not, why not?